src/model/managing.officer.corporate.model.ts

```diff
@@ -19,6 +19,7 @@ export const ManagingOfficerCorporateKeys: string[] = [
 
 export interface ManagingOfficerCorporate {
   id: string;
+  links?: Links;
```

Comment:
I noticed Olayode is not including the links in the BO PR. Could you remove it for now?